test_reproduction_cases.py

```
@@ -0,0 +1,95 @@
+        
+        # This was failing with planning=True due to manual approval requirement
+        agents = AgentTeam(
+            agents=[writer, editor], 
+            memory=True, 
+            planning=True  # This should now auto-approve in non-interactive mode
+        )
+        
+        print("✅ AgentTeam with planning=True created successfully")
+        print("✅ Bug 2 (planning approval) appears fixed for non-interactive environments")
+        
+    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)